src/GraphRequest.ts

```diff
@@ -199,38 +199,40 @@ export class GraphRequest {
 
     delete(callback?:GraphRequestCallback):Promise<any> {
         let url = this.buildFullUrl();
-        return this.sendRequestAndRouteResponse(request.del(url), callback)
+        return this.sendRequestAndRouteResponse(
+            new Request(url, { method: RequestMethod.DELETE, headers: new Headers() }),
```

Comment:
There are always headers present
        headers.append('Authorization', 'Bearer ' + accessToken);
        headers.append('SdkVersion', "graph-js-" + packageInfo.version);

But I will change the code so that it is a bit more straightforward